gemmbench/gemm_bench.py

```diff
@@ -76,10 +76,16 @@ def compile_gemm(tag, config, kernel_dir, vmfb_dir, target, extra_compiler_args,
         default=None,
         help="Directory to which executable files will be dumped."
     )
+    parser.add_argument(
+        "--raw_accumulators",
+        type=bool,
+        default=False,
```

Comment:
Should we use `action=store_true` to make true be the default when the flag is set? I think that's the standard practice when using argparse